src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Colors {
+    White,
+    Blue,
+    Black,
+    Green,
+    Grey,
+    Pink,
+    Yellow,
+    Orange,
+    Purple,
+    Red,
+    Brown
+}
```

Comment:
1. Singular name for class
2. Please follow enum values naming convention
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html